From ccf40d7dd5205e19d091a95ce6da17360c4b6b0f Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@toradex.com>
Date: Mon, 14 Jun 2021 20:30:59 +0300
Subject: [PATCH 1/5] verdin-imx8mm: use TEZI distro boot script

The distro boot script in TEZI image names boot-tezi.scr.
Enable using this particular name of script for u-boot images
which are used as a part of TEZI.

Related-to: TEI-791
Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@toradex.com>
---

 configs/verdin-imx8mm_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/verdin-imx8mm_defconfig b/configs/verdin-imx8mm_defconfig
index 0211bf2197..6cef60851d 100644
--- a/configs/verdin-imx8mm_defconfig
+++ b/configs/verdin-imx8mm_defconfig
@@ -13,6 +13,7 @@ CONFIG_SYS_I2C_MXC_I2C3=y
 CONFIG_ENV_SIZE=0x2000
 CONFIG_ENV_OFFSET=0xFFFFDE0
 CONFIG_DM_GPIO=y
+CONFIG_TDX_EASY_INSTALLER=y
 CONFIG_TARGET_VERDIN_IMX8MM=y
 CONFIG_SPL_MMC_SUPPORT=y
 CONFIG_SPL_SERIAL_SUPPORT=y
-- 
2.31.1

